crates/core/tedge/src/cli/connect/c8y_direct_connection.rs

```diff
@@ -18,6 +19,7 @@ use rumqttc::Transport;
 
 // Connect directly to the c8y cloud over mqtt and publish device create message.
 pub fn create_device_with_direct_connection(
+    use_basic_auth: bool,
```

Comment:
To we need to create the device on Cumulocity when the device is authenticated using a password? Somehow the device should already been known by the tenant.